### PR TITLE
LEARNER-2594 course names discrepancy for coupon redemption

### PR DESCRIPTION
**Pull request by Ayub-Khan**
[**LEARNER-2594**](https://openedx.atlassian.net/browse/LEARNER-2594)
Updated code to use the course title from the discovery. If unable to find it than use the old one.

### CODE REVIEW

**Comment by schenedx** [comment]
@mjfrey Please review

File: ecommerce/extensions/api/v2/views/vouchers.py [hunk 1]
```diff
@@ -286,6 +286,6 @@ def get_course_offer_data(
             'credit_provider_price': credit_provider_price,
             'seat_type': product.attr.certificate_type,
             'stockrecords': serializers.StockRecordSerializer(stock_record).data,
-            'title': course.name,
+            'title': course_info.get('title', course.name),
```
Comment (schenedx) [inline ecommerce/extensions/api/v2/views/vouchers.py:289]: Should we default back to `course.name` if the title property has no value in `course_info`?

**Review by schenedx — COMMENTED** [review]
Got a question here

File: ecommerce/extensions/api/v2/views/vouchers.py (hunk 1 above)
Comment (Ayub-Khan) [reply ecommerce/extensions/api/v2/views/vouchers.py:289]: I think if we do not have any title its better to show course.name instead of empty string .

**Review by schenedx — APPROVED** [review]
👍

**Review by mjfrey — APPROVED** [review]

**Pull request merged**